.github/workflows/main.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - uses: subosito/flutter-action@v1
         with:
           channel: ${{ matrix.flutter_version }}
-      - run: ./tool/flutter_ci_script.sh
+      - run: source tool/flutter_ci_script.sh
```

Comment:
Can you give me a little detail on this particular change? I know `source` changes whether the script is run in a new process or not, but beyond that I don't know the effect.

Reply:
If I remember correctly, I had to use `source` to get Actions to recognize the non-zero exit code as a failure. 

Using`./` runs a script in a separate shell, whereas `source` runs the script in the current shell session, so I was able to work around it using `source`. 

I think this may have been an issue with Actions, so I reverted this change in 8f4a365 and the failures on dev and beta seem to be reporting correctly now.